### PR TITLE
RUMM-741 Provide SQLiteDatabase#transactionTraced extension function

### PR DESCRIPTION
**Pull request by mariusc83**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

File: dd-sdk-android-ktx/README.md [hunk 1]
```diff
@@ -121,6 +121,17 @@ If you're using Kotlin Coroutine Flow, you can propagate Flow errors to your RUM
     }
 ```
 
+#### Tracing SQLite transaction
+
+If you are using SQLiteDatabase to persist your data locally, you can trace the database transaction using the following method:
```
Comment (ruthnaebeck) [inline dd-sdk-android-ktx/README.md:126]:
```suggestion
If you are using SQLiteDatabase to persist data locally, you can trace the database transaction using the following method:
```

File: dd-sdk-android-ktx/README.md [hunk 2]
```diff
@@ -121,6 +121,17 @@ If you're using Kotlin Coroutine Flow, you can propagate Flow errors to your RUM
...
+
+```kotlin
+   sqliteDatabase.transactionTraced("<SPAN_NAME>",isExclusive){
+        // Your queries here
+   } 
+```
+It behaves as the `SQLiteDatabase.transaction` method provided in the `core-ktx` AndroidX package and only requires a span operation name.
```
Comment (ruthnaebeck) [inline dd-sdk-android-ktx/README.md:133]:
```suggestion
It behaves like the `SQLiteDatabase.transaction` method provided in the `core-ktx` AndroidX package and only requires a span operation name.
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]
docs review

**Review by xgouchet — APPROVED** [review]
Mostly naming and KDoc comments 👍

**Pull request merged**